### DISCUSSION TITLE
ImportError: Cannot import 'xyz_to_heremap' from 'leafmap.basemaps'

### DISCUSSION THREAD

**Discussion by duynguyen02** [opengeos/leafmap#1012] [Q&A] [answered]
Hello,  

I’m encountering an error while using the `leafmap` library with Here Maps. Here is the error message I received:  

```python
ImportError: cannot import name 'xyz_to_heremap' from 'leafmap.basemaps'
```
The error occurs on the line `from .basemaps import xyz_to_heremap` in the file `leafmap/heremap.py`.  

Also, could it be that `xyz_to_heremap` hasn’t been implemented yet?

Could anyone help me understand the cause and provide a solution for this issue?  

Thank you very much!

**Comment by giswqs** [comment, marked as answer]
There heremap module has been deprecated. I would discourage using it.
